### ISSUE TITLE
Make mechanisms useful and add them to ctf and tth

### ISSUE THREAD

**Issue by Shadowblitz16** [transhumandesign/kag-base#497]
So right now mechanisms are weak don't have much range and are easy to destroy 

I have a few suggestions on how to fix this..
- Make all mechanisms including wires have same health and defense and stone blocks
  this makes them actually worth using and not able to be ripped apart like paper
- Make wires reach 16 tiles instead of 10
  this allows allows for longer larger contraptions more quickly
- Make a equivalent of a repeater in which the it refreshes the signal and delays it by 1
  this makes building mechanisms easier and also puts some sort of control on them
- I would then suggest that they are added to ctf and tth for builders to learn and use 
  this makes gameplay more fun and in turn increases the lifetime of the game

things to note..
- mechanisms might need a little bit less then the stone blocks health but should still be as resilient to bombs and bomb arrows

**Comment by xXIlluminatiPartyXx** [comment]
Yes because this may increase the lifetime of the game by adding diversity

**Comment by Vam-Jam** [comment]
I'd like to see this get used, currently a lot of hard work has been put into making mechanisms possible, but they haven't been used outside of sandbox (which little people play)

They need to get tweaked a little before being added into CTF or TTH, I remember a while ago they would just stop working for no reason upon next map (not sure if this still happens)

Currently they need a guide to explain how they work, along with a few QOL changes like allowing knights to slash the wire, and maybe more offensive/defensive options, such as fire traps and or stun traps.

**Comment by Shadowblitz16** [comment]
If knight are able to slash the wire they should be at least two double slashes to cut.
this makes the wires somewhat worth placing and gives builders a bit of leniency to learn and build.

but I agree with you. maybe wires could be placed over blocks at the cost of being able to access them to repair them if they get damaged.
